CHANGELOG.md

```diff
@@ -1,5 +1,89 @@
 # osquery Changelog
 
+<a name="5.0.2"></a>
+## [5.0.2](https://github.com/osquery/osquery/releases/tag/5.0.2)
+
+[Git Commits](https://github.com/osquery/osquery/compare/4.9.0...5.0.2)
+
+Representing commits from 21 contributors! Thank you all.
+
+osquery 5.0 is a tremendously exciting release.
+* We now install into /opt/osquery on macOS and Linux for better portability.
+* Our default and recommended installation for macOS uses an application bundle to support entitlement-based features.
+* We now use Endpoint Security APIs for various event-based tables on macOS (more to come in the future!)
+* We now use an osquery-organization macOS code signing certificate.
```

Comment:
Either here or in breaking changes, "Reminder: organizations using MDM to manage osquery should note the impact of the code-signing identity change to their deployments." and a link to the docs https://osquery.readthedocs.io/en/latest/deployment/process-auditing/#automatically-granting-permissions-silent-installs